tests/akash/matrix.futil.data

```diff
@@ -0,0 +1,26 @@
+{
+  "A": {
+    "data": [[1,0,4,6],[2,5,0,3],[1,2,3,5],[2,1,2,3]],
+    "format": {
+      "numeric_type": "bitnum",
+      "is_signed": false,
+      "width": 32
+    }
+  },
+  "B": {
```

Comment:
For readability and easy comparison with the .expect file, consider formatting this file like the .expect file. At least on my computer, this is just a matter of telling VSCode that the file is a JSON file and then saving.